flocx_market/cmd/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, request
+from flask_restful import Resource, Api, reqparse
+
+from resources.offer import Offer, OfferList
+
+app = Flask(__name__)
+api = API(app)
+
+api.add_resource(Offer, '/offer/<string:id>')
+api.add_resource(OfferList, '/offers')
+
+if __name__ == '__main__':
+    app.run(debug = True)
```

Comment:
This should probably be renamed api.py, and call a function in flocx_market/api.py. Lines 6-10 should be there, and not here.

Reply:
I accidentally left that file in there. I haven't used app.py for anything. Everything runs through api.py